dbfit-java/build.gradle

```diff
@@ -290,10 +290,71 @@ project('teradata') {
 project('netezza') {
     description = 'DBFit netezza engine'
 
+    def propertiesFile = 'TestDbConnectionDbFitNetezza.properties'
+    def jdbcDriverClass = 'org.netezza.Driver'
+    def jdbcUrlPrefix = 'jdbc:netezza://'
+    def jdbcUrl
+    def database
+    def userName
+    def passWord
+    def delimiterType = 'row'
+
     dependencies {
         compile project(':dbfit-java:core')
         runtime 'org.netezza:nzjdbc3:7.2@jar'
     }
+
+    task setConnectionProperties {
+        doLast {
+            def props = new Properties()
+            file(propertiesFile).withInputStream { props.load(it) }
+            userName =  props.getProperty('username')
+            passWord =  props.getProperty('password')
+            def service = props.getProperty('service')
+            def serviceParts = service.tokenize(':')
+            def serviceHost = serviceParts[0]
+            def servicePort = serviceParts[1]
+            def propDatabase = props.getProperty('database')
+
+            database = propDatabase.length() > 0 ? ('/' + propDatabase) : ''
+            jdbcUrl = jdbcUrlPrefix + serviceHost + ((servicePort.length() > 0) ? (':' + servicePort) : '')
+        }
+    }
+
+    integrationTest.dependsOn setConnectionProperties
+
+    integrationTest.doFirst {
+        copy {
+            from propertiesFile + '.default'
+            into '.'
+            rename '(.*).default', '$1'
+        }
+
+        copy {
+            from propertiesFile + '.custom'
+            into '.'
+            rename '(.*).custom', '$1'
+        }
+
+        dbdeploy(jdbcDriverClass, jdbcUrl + database, userName, passWord, delimiterType)
+    }
+
+    task createTestDatabase {
+        doLast {
+            ant.sql(
+                classpath: configurations.runtime.asPath,
+                driver: jdbcDriverClass,
+                url: jdbcUrl + '/system',
+                userid: userName,
+                password: passWord,
+                delimiterType: delimiterType,
+                autocommit: true,
+                src: "${rootProject.projectDir}/test_vm/scripts/netezza/sql/create-databases-netezza.sql"
+            )
+        }
+    }
+
+    createTestDatabase.dependsOn setConnectionProperties
```

Comment:
Ah, I see now that we have `TestDbConnectionDbFitNetezza.properties`. The approach for other databases has been to have a `*.properties.default` file instead of `*.properties` since the latter could be replaced by `*.custom` version during the build and I don't quite like touching files which are under version control.